### ISSUE TITLE
Tampermonkey broke Firefox's built in XML document viewer again

### ISSUE THREAD

**Issue by grumble0** [Tampermonkey/tampermonkey#1609]
Go to https://www.w3schools.com/xml/cd_catalog.xml

## Expected Behavior
XML files opened in Firefox should display the XML contents.  For the cd_catalog.xml example, should display a page that starts with (not possible to show styling in github):
```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<CATALOG>
<CD>
<TITLE>Empire Burlesque</TITLE>
<ARTIST>Bob Dylan</ARTIST>
<COUNTRY>USA</COUNTRY>
<COMPANY>Columbia</COMPANY>
<PRICE>10.90</PRICE>
<YEAR>1985</YEAR>
</CD>
<CD>
<TITLE>Hide your heart</TITLE>
<ARTIST>Bonnie Tyler</ARTIST>
```

## Actual Behavior
XML tags are stripped and only the element contents are displayed.  For the cd_catalog.xml example the page starts with:
```
Empire Burlesque Bob Dylan USA Columbia 10.90 1985 Hide your heart Bonnie Tyler
```

## Specifications
Firefox 105.0.2
TM v4.18.0

## Script
No scripts required.

## Notes
Perhaps similar to https://github.com/Tampermonkey/tampermonkey/issues/1480 although the sandbox mode workaround there does not work for this issue.
Downgrading to TM v4.17.6161 fixes the issue.

**Comment by derjanb** [comment]
> although the sandbox mode workaround there does not work for this issue

Have you saved the settings after switching to "JavaScript+DOM" or "Force JavaScript" mode?

**Comment by grumble0** [comment]
> > although the sandbox mode workaround there does not work for this issue
> 
> Have you saved the settings after switching to "JavaScript+DOM" or "Force JavaScript" mode?

No!  Sorry, I didn't see the Save button at the bottom of each section.  I expected any Save button to be at the top or bottom of the page, didn't see one, so assumed the setting was applied immediately.

Retested with saving... these Sandbox modes fix the bug:
- Force JavaScript
- JavaScript+DOM
- Force DOM

**Comment by derjanb** [comment]
No problem. 

Closing as duplicate of #1480

**Comment by Kyogre** [comment]
Same for `wsdl` links. Setting Sandbox Mode to `JavaScript+DOM` in advanced config mode fixed the problem.

**Comment by Kyogre** [comment]
Yeah, it is successfully fixed in beta 4.19.0 with default sandbox. I already tested. I just realized that inside wsdl page there is just xml scheme 😉
[Example wsdl link](http://www.thomas-bayer.com/axis2/services/BLZService?wsdl).
